R/utils.R

```diff
@@ -210,6 +210,118 @@ get_currency_str <- function(currency,
   }
 }
 
+resolve_footnote_placement <- function(data,
+                                       colname,
+                                       rownum,
+                                       input_placement,
+                                       cell_content,
+                                       context) {
+
+  if (input_placement %in% c("left", "right")) {
+    return(input_placement)
+  }
+
+  cell_alignment <-
+    get_alignment_at_body_cell(
+      data = data,
+      colname = colname,
+      rownum = rownum,
+      context = context
+    )
+
+  # If cell alignment has been set then always choose 'right' placement
+  if (!is.null(names(cell_alignment)) && names(cell_alignment) == "user_set") {
+    return("right")
+  }
+
+  if (cell_alignment == "right") {
```

Comment:
Let's just say auto means mark placement is opposite of cell alignment.

Reply:
Done. However, now there needs to be some followup work done to get the alignments of the summary cells.

Reply:
Done.